pytorch_blade/bazel/torch_mlir/BUILD

```diff
@@ -0,0 +1,49 @@
+load("@llvm-project//mlir:tblgen.bzl", "gentbl_cc_library", "gentbl_filegroup", "td_library")
+
+package(
```

Comment:
Does torch MLIR community  support bazel? Do we have to maintain our own bazel build file for torch-mlir?